share/functions/fish_default_key_bindings.fish

```diff
@@ -2,7 +2,9 @@
 function fish_default_key_bindings -d "Default (Emacs-like) key bindings for fish"
 	if not set -q argv[1]
 		if test "$fish_key_bindings" != "fish_default_key_bindings"
-			set -g fish_key_bindings fish_default_key_bindings # This triggers the handler, which calls us again and ensures the user_key_bindings are executed
+			# Allow the user to set the variable universally
+			set -q fish_key_bindings; or set -g fish_key_bindings
```

Comment:
Interesting. No objection but this should be mentioned wherever this var is documented.

P.S., I was thinking of proposing doing the same thing for cd history (i.e., `$dirprev`) in light of feedback I got on my `mcd` command proposal.